### DISCUSSION TITLE
Add the possibility Tunnelblick's xor patch on openvpn

### DISCUSSION THREAD

**Discussion by dtony974** [pivpn/pivpn#1761] [Feature requests]
Hello do you think is possible to use the package of https://github.com/cuscoboy/openvpn-xor, to have the possible to become more hide ?
Or is possible to just add the line in the server.conf of openvpn: scramble obfuscate yourpassword

Regard,

**Comment by coolapso** [comment]
No. its beyond the scope of PiVPN, openvpn official is a  trusted, well known and established software, audited by third parties with well known and established  release channels widely available on the distributions repositories. Using forks maintained by third parties brings a not only a lot of overhead maintenance issues as well as trust issues! therefore its not reliable and we want to make sure that pivpn users are offered with something they can trust!  

And thinking about trust I don't even advise using the packaged you have linked. All we get is the compiled debian package, how can we trust that patch doesn't bring some poison pills with it?  how is the package built? Based on what source code? That's a big No No.
